models/nomiswap/bnb/nomiswap_bnb_trades.sql

```diff
@@ -0,0 +1,107 @@
+{{ config(
+    alias = 'trades'
+    ,partition_by = ['block_date']
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    ,post_hook='{{ expose_spells(\'["bnb"]\',
+                                      "project",
+                                      "nomiswap",
+                                    \'["codingsh"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2022-08-31' %}     -- select min(evt_block_time) from nomiswap_bnb.NomiswapPair_evt_Swap
```

Comment:
I modify this date based on [the query result](https://dune.com/queries/1651513?d=1)